BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -346,6 +349,26 @@ - (void)setSession:(BOXAbstractSession *)session
     self.queueManager.session = self.session;
 }
 
+- (void)setTempCacheDir:(NSString *)folderPath
```

Comment:
Minor: I'm partial to avoiding abbreviations in method/variable names, but it's fine.